Examples/zib-FLACCPainScale-01.xml

```diff
@@ -137,22 +137,6 @@
             </coding>
         </valueCodeableConcept>
     </component>
-    <component>
-        <code>
-            <coding>
-                <system value="http://loinc.org"/>
-                <code value="38219-2"/>
-                <display value="Pain severity Cry FLACC"/>
-            </coding>
-        </code>
-        <valueCodeableConcept>
-            <coding>
-                <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.21.3"/>
-                <code value="Act1"/>
-                <display value="Kronkelt, wringt zich in bochten, verandert steeds van houding,is gespannen"/>
-            </coding>
-        </valueCodeableConcept>
-    </component>
```

Comment:
Kan niet zo goed inschatten waarom dit weggehaald wordt terwijl er niks voor in de plek wordt gezet